fuzz/FuzzingUtils.h

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 Mikhail Khachaiants <mkhachaiants@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef URI_FUZZING_UTILS_H
+#define URI_FUZZING_UTILS_H 1
+
+#include "uriparser/Uri.h"
+#include <fuzzer/FuzzedDataProvider.h>
+#include <string>
+
+
+
+using String = std::basic_string<URI_CHAR>;
+
+
+
+inline String consumeBytesAsString(FuzzedDataProvider & fdp, size_t len) {
+	String str(len, 0);
+	size_t bytes = fdp.ConsumeData(str.data(), str.length() * sizeof(URI_CHAR));
```

Comment:
```suggestion
	size_t bytes = fdp.ConsumeData(str.data(), len * sizeof(URI_CHAR));
```
Would seem simpler if identical. Just an idea.

Reply:
I've refactored FuzzingUtils.h a little bit and applied your suggestion